packages/edit-site/src/components/global-styles/font-library-modal/installed-fonts.js

```diff
@@ -50,8 +51,12 @@ function InstalledFonts() {
 		notice,
 		setNotice,
 		fontFamilies,
+		deactivateFontFamily,
+		activateFontFamily,
 	} = useContext( FontLibraryContext );
 	const [ isConfirmDeleteOpen, setIsConfirmDeleteOpen ] = useState( false );
+	const [ isSelectAllChecked, setIsSelectAllChecked ] = useState( true );
```

Comment:
We should be able to determine ad hoc whether all fonts are selected or not without using state. Additionally, I think it's necessary to have a state that shows that only some fonts are selected.

Below is a screenshot of the DataView:

![dataview](https://github.com/user-attachments/assets/da3cb075-72dd-4210-98f0-e4529105d1a7)